src/CLI/Options.cs

```diff
@@ -31,5 +31,8 @@ public class Options : IOptions
 
         // CommandLineParser will never set this value!
         public VerbosityLevel VerbosityLevel { get; set; } = VerbosityLevel.Default;
+
```

Comment:
Can we keep VerbosityLevel as the last member, since it's different than the othes?